backend/app.ts

```
@@ -41,6 +41,16 @@ app.use(
   })
 )
 
+app.post('/subscribe', (req, res) => {
+  let subscription: Subscription = req.body
+  if (subscription && subscription.endpoint) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)